apps/editor/src/editorCore.ts

```diff
@@ -281,7 +283,9 @@ class ToastUIEditorCore {
     }
 
     this.eventEmitter.emit('load', this);
-    this.moveCursorToStart();
+    if (this.options.autofocus) {
+        this.moveCursorToStart();
```

Comment:
In this case, not only the focus but also the cursor position is affected. The operation for cursor position should remain the same, so that only the focus can be controlled as shown below.
```js
this.moveCursorToStart(this.options.autofocus);

// ...

moveCursorToStart(focus = true) {
  this.getCurrentModeEditor().moveCursorToStart(focus);
}
```
And you're gonna have to change the `base.ts`, too.
Can you work on it this way? I'd appreciate it if you could do the same thing with `moveCursorToEnd` API!

Reply:
Here it is.